hazelcast/src/test/java/com/hazelcast/wan/impl/WanNoDelayReplicationTest.java

```diff
@@ -0,0 +1,57 @@
+package com.hazelcast.wan.impl;
+
+import com.hazelcast.map.impl.wan.MapReplicationUpdate;
+import com.hazelcast.test.AssertTask;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.wan.ReplicationEventObject;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertSame;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.spy;
+
+/**
+ * @author Henri Tremblay
+ */
```

Comment:
In fact I did that on purpose. Because the other test is a NightlyTest.
Mine is a real unit test.

On 17 September 2015 at 04:11, emrahkocaman <notifications@github.com>
wrote:

> In
> hazelcast/src/test/java/com/hazelcast/wan/impl/WanNoDelayReplicationTest.java
> <https://github.com/hazelcast/hazelcast/pull/6229#discussion_r39720156>:
>
> > +
> > +import com.hazelcast.map.impl.wan.MapReplicationUpdate;
> > +import com.hazelcast.test.AssertTask;
> > +import com.hazelcast.test.HazelcastTestSupport;
> > +import com.hazelcast.wan.ReplicationEventObject;
> > +import org.junit.Test;
> > +
> > +import static org.junit.Assert.assertEquals;
> > +import static org.junit.Assert.assertFalse;
> > +import static org.junit.Assert.assertSame;
> > +import static org.mockito.Mockito.doThrow;
> > +import static org.mockito.Mockito.spy;
> > +
> > +/**
> > + * @author Henri Tremblay
> > + */
>
> Or better, there already is a test class related to wan replication ->
> WanReplicationTest
> <https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/test/java/com/hazelcast/wan/WanReplicationTest.java>
> Could you please move your test cases to that class?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hazelcast/hazelcast/pull/6229/files#r39720156>.
>